src/main/java/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -804,13 +818,13 @@ private Object decodeBasicOther(
                 case 0x92:
                     position.set(Position.KEY_ALARM, Position.ALARM_CORNERING);
                     break;
+                case 0x93:
+                    position.set(Position.KEY_ALARM, Position.ALARM_ACCIDENT);
+                    break;
                 default:
                     position.set(Position.KEY_ALARM, Position.ALARM_GENERAL);
                     break;
             }
-
-            position.set("alarmValue", buf.readShort());
```

Comment:
Wha happened to this value?

Reply:
As I wrote above - it is not there for JC100 at all and is zero for JC400. Only over speed alarm from JC400 has the actual speed value there but you get that from the location update already.
I was wrong in #4412 and read the packet serial number by mistake.